src/main/xar-resources/data/xquery/xquery.xml

```diff
@@ -607,7 +607,13 @@
                     </listitem>
                 </varlistentry>
                 <varlistentry>
-                    <term> <literal>expand-xincludes : ( boolean )</literal> </term>
+                    <term> <literal>output-doctype : ( "yes" | "no" )</literal> </term>
+                    <listitem>
+                        <para>Must the serializer expand XInclude elements?</para>
```

Comment:
This description appears to be unchanged from the `expand-xincludes` explanation.

Reply:
Thanks Joe - not sure what happened. I have fixed it now.

Reply:
Thanks, Adam!